testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -120,14 +124,43 @@ def test_rmsd(self):
         assert_almost_equal(rmsd, 6.820321761927005, 5,
                             err_msg="RMSD calculation between 1st and last "
                             "AdK frame gave wrong answer")
-        #test mass_weighted
+        # test masses as weights
+        last_atoms_weight = self.universe.atoms.masses
+        A = self.universe.trajectory[0]
```

Comment:
I think the timestep is always a new one I think. But the timestep has a similar API as a atomgroup so it can work if the rmsd only looks at the positions.